examples/resources/awscc_eks_cluster/eks_cluster_controlplane_logging.tf

```diff
@@ -0,0 +1,132 @@
+data "aws_partition" "current" {}
+
+locals {
+  policy_arn_prefix = "arn:${data.aws_partition.current.partition}:iam::aws:policy"
+}
+
+variable "eks_default_tags" {
+  description = "Default tags to be applied to EKS resources"
+  default = [
+    {
+      key   = "Environment"
+      value = "Development"
+    },
+    {
+      key   = "Modified By"
+      value = "AWSCC"
+    }
+  ]
+}
+
+variable "eks_cluster_name" {
+  description = "EKS Cluster Name"
+  type        = string
+  default     = "example-cluster"
+}
+
+variable "eks_cluster_version" {
+  description = "EKS Cluster Version"
+  type        = string
+  default     = "1.27"
+}
+
+variable "eks_cluster_subnets" {
+  description = "Subnets for EKS Cluster"
+  type        = list(string)
+  default     = ["subnet-xxxx", "subnet-yyyy"] // Provide a list of subnet-ids for Amazon EKS Cluster
+}
+
+variable "enabled_cluster_log_types" {
+  default     = ["api", "audit", "authenticator", "controllerManager", "scheduler"]
+  description = "A list of the Logs to be enabled in control plane."
+  type        = list(string)
+}
```

Comment:
Same here,
Probably the best is to show your core TF configuration and remove those variables.
In the resource block, use example like
name = "example_cluster"
version = "1.1"

Reply:
@jackywong-amazon This has been taken care.